src/lib.rs

```diff
@@ -301,38 +304,12 @@ decl_module! {
             DispatchClass::Normal,
             Pays::Yes
         )]
-        fn create(origin,
+        fn create(
+            origin,
             threshold: u16,
-            other_signatories: Vec<T::AccountId>
+            other_signatories: Vec<T::AccountId>,
         ) -> DispatchResult {
-            let who = ensure_signed(origin)?;
-            ensure!(threshold >= 1, Error::<T>::ZeroThreshold);
-            let max_sigs = T::MaxSignatories::get() as usize;
-            ensure!(threshold as usize <= other_signatories.len() + 1, Error::<T>::TooFewSignatories);
-            ensure!(other_signatories.len() < max_sigs, Error::<T>::TooManySignatories);
-            let signatories = Self::ensure_sorted_and_insert(other_signatories.clone(), who.clone())?;
-
-            let multi_account_index = MultiAccountIndex::get()
-                .checked_add(1)
-                .expect("multi account indices will never reach 2^64 before the death of the universe; qed");
-
-            let id = Self::multi_account_id(multi_account_index);
-
-            let deposit = T::MultiAccountDepositBase::get()
-                + T::MultiAccountDepositFactor::get() * (other_signatories.len() as u32 + 1).into();
-
-            T::Currency::reserve(&who, deposit)?;
-
-            MultiAccountIndex::put(multi_account_index);
-
-            <MultiAccounts<T>>::insert(&id, MultiAccountData {
-                threshold,
-                signatories,
-                deposit,
-                depositor: who.clone(),
-            });
-
-            Self::deposit_event(RawEvent::NewMultiAccount(who, id));
+            ensure!(false, Error::<T>::DeprecatedPallet);
 
             Ok(())
```

Comment:
```suggestion
            Err(Error::<T>:: DeprecatedPallet.into())
```

With this, we can remove the ensure check above